packages/front-generator/src/generators/react-typescript/entity-cards/template/EntityCards.tsx.ejs

```diff
@@ -24,9 +24,10 @@ export class <%= className %> extends React.Component {
     return (
       <div className='narrow-layout'>
         {items.map(e =>
-          <Card title={e._instanceName}
-                key={e.id}
-                style={{marginBottom: '12px'}}>
+        <% /* todo null check of e.id will be removed after we strict id type in https://github.com/cuba-platform/frontend/issues/119 */ %>
+        <Card title={e._instanceName}
+              key={e.id ? e.id : undefined}
```

Comment:
Could we move it to function, e.g. 
```
getEntityId(Entity.Name, instance)
```
in that case we potentially can return proper id determined by metadata (in that case we should also add Id field info to REST metadata). Also, even if after we add id info in studio we still have to support older studio model format (which does not contain id info) - so we we'll not be able to remove this check